### PR TITLE
bug fixing

### PR DESCRIPTION
**Pull request by skoooch**

### CODE REVIEW

File: src/main/java/FrameworksDrivers/ViewUI.java [hunk 1]
```diff
@@ -1,15 +1,6 @@
 package FrameworksDrivers;
-
```
Comment (czhang0312) [inline src/main/java/FrameworksDrivers/ViewUI.java:2]: Nice clean up here

**Review by czhang0312 — APPROVED** [review]
lgtm

**Review by vvickychan — APPROVED** [review]
The code cleanups look  good to me

**Comment by carolinexuxy** [comment]
All changes look good!

**Review by carolinexuxy — APPROVED** [review]

**Pull request merged**